docs/Tutorials/Privacy/Configuring-Privacy.md

```diff
@@ -8,8 +8,8 @@ Configuring a network that supports private transactions requires starting an Or
 Hyperledger Besu node. Besu command line options associate the Besu node with the Orion node.
 
 This tutorial assumes you have completed setting up an IBFT 2.0 network to the point where you have
-[created the genesis file](../Private-Network/Create-IBFT-Network.md#5-create-genesis-file). If
-not, complete steps 1 to 5 of the
+[created the genesis file](../Private-Network/Create-IBFT-Network.md#5-copy-the-node-private-keys-to-the-node-directories).
```

Comment:
Perhaps change this to:
`[created the genesis file and copied the private keys]`